ci-operator/config/openstack-k8s-operators/repo-setup/openstack-k8s-operators-repo-setup-master.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
We need to check which kind of jobs make sense to add here.
https://github.com/openstack-k8s-operators/repo-setup has no Makefile targets, and no golang code there.